README.md

```diff
@@ -1 +1,72 @@
-# quest_maker
+## About this application
+QuestMaker is a web application that integrates with FitBit and makes exercise more rewarding by enabling users to easily create and participate in quests with their friends. In its simplest form, a quest is a series of milestones that users reach as they cover more distance over time. For users with FitBits, daily progress is updated automatically; for those without, manual entry is easy.
+
+The idea for this application comes from the [Walk to Rivendell Challenge](http://home.insightbb.com/~eowynchallenge/index.html), which provides a [series of waypoints](http://home.insightbb.com/~eowynchallenge/Tools/Bag_end/bag_end.html) for Frodo's walk across Middle Earth. Since its inception in 2003, many people have made the whimsical trek across Tolkien's Middle Earth by manually charting their mileage each day and sharing their progress with friends. QuestMaker personalizes and automates this process, allowing you to focus your energies on outwalking the rest of your ~~friends~~ ~~competitors~~ fellowship!
+
+
+## Getting started
+See [Getting Started](GettingStarted.md) for detailed info on how to set everything up on your machine.
+
+
+## Running the test suite
+TODO
+
+
+## Git Workflow
+
+### To work on the app:
+See [Getting Started](GettingStarted.md) for initial setup. Once you are set up, follow the following procedure to contribute code.
+
+#### A note on branch names:
+* The production branch is ``master``.
+* The development branch is ``develop``.
+* Feature branches branch off from ``develop`` and are named ``feature-<description>``, e.g. feature-admin-editing.
+* Hotfix branches are named ``hotfix-<description>``;  they may branch from develop or master and should be merged back into both.
+
+Thus, if you are working on a **new feature**, follow this procedure:
+
+First, pull down from develop (to make sure you're up-to-date) and create a new branch:
+  ```
+  git pull origin develop
```

Comment:
You should add ``git checkout develop`` at the beginning of this code block to emphasize that feature branches are branched off of develop.